--- lwinput.c	2022-03-01 22:25:30.842360400 +0900
+++ b/lwinput.c	2022-03-20 04:38:01.428588000 +0900
@@ -47,6 +47,7 @@
 
 #define MPEG4_FILE_EXT      "*.mp4;*.m4v;*.m4a;*.mov;*.qt;*.3gp;*.3g2;*.f4v;*.ismv;*.isma"
 #define INDEX_FILE_EXT      "*.lwi"
+#define FFMPEG_FILE_EXT     "*.aac;*.ac3;*.aiff;*.apng;*.avi;*.ico;*.m2t;*.m2ts;*.mkv;*.mp3;*.ogg;*.opus;*.ts;*.webm;*.wmv"
 #define ANY_FILE_EXT        "*.*"
 
 static char plugin_information[512] = { 0 };
@@ -79,6 +80,7 @@
     "L-SMASH Works File Reader",                                    /* Name of plugin */
     "MPEG-4 File (" MPEG4_FILE_EXT ")\0" MPEG4_FILE_EXT "\0"        /* Filter for Input file */
     "LW-Libav Index File (" INDEX_FILE_EXT ")\0" INDEX_FILE_EXT "\0"
+    "FFmpeg Supports File (" FFMPEG_FILE_EXT ")\0" FFMPEG_FILE_EXT "\0"
     "Any File (" ANY_FILE_EXT ")\0" ANY_FILE_EXT "\0",
     "L-SMASH Works File Reader r" LSMASHWORKS_REV "\0",             /* Information of plugin */
     func_init,                                                      /* Pointer to function called when opening DLL (If NULL, won't be called.) */
